code/game/machinery/computer/buildandrepair.dm

```diff
@@ -219,6 +219,9 @@
 /obj/item/circuitboard/prisoner
 	name = "Circuit board (Prisoner Management)"
 	build_path = /obj/machinery/computer/prisoner
+/obj/item/circuitboard/brigcells

```

Comment:
spacing between this and the one above it.

Reply:
None of these have spacing and spacing just one of them will look wierd

Reply:
I'd probably put spaces between all of them just because it's an eyesore